src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/DraftController.java

```diff
@@ -69,4 +71,22 @@ public ResponseEntity create(
 
         return status(HttpStatus.CREATED).build();
     }
+
+    @PutMapping(path = "/{id}")
+    public ResponseEntity update(
+        @PathVariable int id,
+        @RequestHeader(AUTHORIZATION) String authHeader,
+        @RequestBody @Valid UpdateDraft updatedDraft
+    ) {
+        String currentUserId = userIdService.userIdFromAuthToken(authHeader);
+
+        return draftRepo
+            .read(id)
+            .filter(d -> Objects.equals(d.userId, currentUserId))
+            .map(d -> {
+                draftRepo.update(id, currentUserId, updatedDraft);
+                return status(HttpStatus.NO_CONTENT).build();
```

Comment:
no content or the updated draft?

Reply:
No content. That's what v2 returns as well.  

In the guide:
>successful PUT requests will usually generate 200 or 204 (if the resource was updated - with or without actual content returned)

[https://hmcts.github.io/restful-api-standards/#put](https://hmcts.github.io/restful-api-standards/#put)

Reply:
ok

Reply:
I think we could update the draft in one go to the DB, i.e. without reading it. What do you think?

Reply:
change to return 403, so separate read is needed